man/paru.conf.5

```diff
@@ -1,53 +1,55 @@
 '\ t
-.TH "PARU.CONF" "5" "2019\-10\-21" "paru v0.0.1" "Paru  Manual"
+.TH "PARU.CONF" "5" "2020\-10\-28" "paru v1.0.0" "Paru Manual"
 .nh
 .ad l
 .SH NAME
-PARU \- AUR Helper
+paru.conf \- paru configuration file
 
 .SH SYNOPSIS
-/etc/paru.conf
-.sp
-~/.config/paru/paru.conf
+$XDG_CONFIG_HOME/paru/paru.conf, $HOME/.config/paru/paru.conf, /etc/paru.conf
 
 .SH DESCRIPTION
-Paru's config file. Based on the format used by \fBpacman.conf (5)\fR.
+Paru's config file. Based on the format used by 
+.BR pacman.conf (5)
 
-Paru will attempt to read the config file from the user's home directory. If that does not exist
-paru will instead attempt to read the file from the system config directory.
+Paru will attempt to read the config file from $XDG_CONFIG_HOME/paru/paru.conf. If
+$XDG_CONFIG_HOME is unset, it will attempt to read the file from 
+$HOME/.config/paru/paru.conf. If neither of those exist, it will read from the
```

Comment:
You're free to edit /etc/paru.conf, so I wouldn't say "it will read from the defaults at /etc/paru.conf".